app/scripts/modules/core/pipeline/config/triggers/docker/dockerTrigger.module.js

```diff
@@ -48,15 +48,20 @@ module.exports = angular.module('spinnaker.core.pipeline.trigger.docker', [
       }
       let key = `${trigger.registry}/${trigger.repository}`;
       $scope.tags = $scope.repositoryMap[key] || [];
-      if ($scope.tags.indexOf(trigger.tag) < 0) {
-        trigger.tag = null;
-      }
     }
 
     this.clearTag = function() {
       trigger.tag = null;
     };
 
+    $scope.getTags = function(search) {
+      var newOne = $scope.tags.slice();
+      if (search && $scope.tags.slice().indexOf(search) === -1) {
+        newOne.unshift(search);
```

Comment:
Its not actually adding anything to the dropdown list, since its not doing the unshift or push to $scope.tags.  Im not great with javascript either, so I can do it another way if someone has a better suggestion.